Sources/SafeDICore/Visitors/InstantiableVisitor.swift

```diff
@@ -317,29 +317,24 @@ public final class InstantiableVisitor: SyntaxVisitor {
         }
     }
 
-    // MARK: Internal
-
-    var instantiable: Instantiable? {
-        guard let instantiableType else { return nil }
+    public var instantiables: [Instantiable] {
```

Comment:
now extensions can create _multiple_ `Instantiable` types, since `Container<Int>` and `Container<String>` are considered distinct Instantiable types by SafeDI, even though they are still a single generic type to the Swift compiler.